### PR TITLE
fix(MRF-11255): add platform field to metrics.json schema

### PR DESCRIPTION
**Pull request by joselanderosmrf**
add platform field to metrics.json schema to fix maduradas build

BREAKING CHANGE: metrics.json requires platform field for custom trackers

### CODE REVIEW

**Comment by joselanderosmrf** [comment]
retest this please

**Comment by joaquimsantmrf** [comment]
retest this please

File: json-lint/schemas/metrics/providers/custom.json [hunk 1]
```diff
@@ -58,10 +79,17 @@
                     },
                     "pixelSrc": {
                         "type": "string"
+                    },
+                    "platforms": {
+                        "type": "array",
+                        "items": {
+                            "type": "string",
+                            "enum": ["AMP"]
```
Comment (joaquimsantmrf) [inline json-lint/schemas/metrics/providers/custom.json]: only AMP?, no TOUCH?

File: json-lint/schemas/metrics/providers/custom.json [hunk 2]
```diff
@@ -16,9 +16,16 @@
                     },
                     "params": {
                         "type": "object"
+                    },
+                    "platforms": {
```
Comment (joaquimsantmrf) [inline json-lint/schemas/metrics/providers/custom.json]: something like in inventory/Adservers/types

"platforms" : {  "$ref": "./types/platformAmpTouch.json" }

File: json-lint/schemas/metrics/main.json [hunk 3]
```diff
@@ -20,6 +20,13 @@
                     },
                     "iframeMessage": {
                         "type": "string"
+                    },
+                    "platforms": {
+                        "type": "array",
+                        "items": {
+                            "type": "string",
+                            "enum": ["AMP"]
```
Comment (joaquimsantmrf) [inline json-lint/schemas/metrics/main.json]: only AMP?

File: json-lint/schemas/metrics/main.json [hunk 4]
```diff
@@ -189,7 +238,38 @@
...
+                        "description": "uaCode",
+                        "pattern": "^(UA-[0-9]{6,}-[0-9]+)$",
+                        "title": "uaCode",
+                        "type": "string"
+                    },
+                    "platforms": {
+                        "type": "array",
+                        "items": {
+                            "type": "string",
+                            "enum": ["AMP"]
```
Comment (joaquimsantmrf) [inline json-lint/schemas/metrics/main.json]: only AMP?

File: json-lint/schemas/metrics/providers/custom.json (hunk 1 above)
Comment (joselanderosmrf) [reply json-lint/schemas/metrics/providers/custom.json]: yes

File: json-lint/schemas/metrics/main.json (hunk 3 above)
Comment (joselanderosmrf) [reply json-lint/schemas/metrics/main.json]: yes, only supported by amp

**Pull request merged**